lib/rules/call-super-in-init.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const ember = require('../utils/ember');
+const utils = require('../utils/utils');
+
+//----------------------------------------------
+// General rule - Call _super in init lifecycle hooks
+//----------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Enforces super calls in init hooks',
+      category: 'General',
+      recommended: true
```

Comment:
Let's set this to `false` so that we can make a minor release and we'll bulk set those recommendations later on in one major release.